apps/journeys-admin-e2e/src/pages/journey-level-actions-page.ts

```
@@ -307,14 +310,26 @@
       await this.page
         .locator("div[class *='MuiAutocomplete-popper'] li")
         .last()
-        .scrollIntoViewIfNeeded()
+        .waitFor({ state: 'attached' })
+      // eslint-disable-next-line playwright/no-wait-for-timeout
```

ESLint

[error] Definition for rule 'playwright/no-wait-for-timeout' was not found.

(playwright/no-wait-for-timeout)